tests/system/action/projector_countdown/test_update.py

```diff
@@ -44,6 +44,25 @@ def test_update(self) -> None:
         assert model.get("default_time") == 30
         assert model.get("countdown_time") == 20
 
+    def test_update_same_title_in_same_id(self) -> None:
+        response = self.request(
+            "projector_countdown.update",
+            {
+                "id": 2,
+                "title": "test",
+                "description": "good bla",
```

Comment:
Please remove the 4 following fields since they are not in the scope of this test.

Reply:
I removed the 4 fields from the test. And will merge it now.